### PR TITLE
Port installer-help page to Fluent (Fixes #8916)

### LINKED ISSUE #8916: [26 May] Port new installer-help template to Fluent
## Description

The new installer-help page added in https://github.com/mozilla/bedrock/pull/8791 has not yet been exposed for string translation in .lang. 

As such, we'd like to graduate this to be translated using Fluent instead - with an aim to have a PR ready to merge on 26th May (shortly after the new system goes live). This will be a good real world test before we tackle WNP78.

This is a good candidate for another front-end dev other than myself to try.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] New installer help page is live in production using Fluent.
- [ ] Strings are exposed via automation on merge to master.

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] There's a risk there may be some bugs in the system still, but that's the job of this test to identify.

#### Tasks
- [x] Create en .ftl file.
- [x] Convert template to use fluent IDs.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
The redesigned installer-help template was never exposed for L10n, so we're going to use it here as a first full page, end-to-end test of the Fluent pipeline. We want to try and get this merged on or before Tuesday 26th May, giving us as much time to test the process as possible, before we begin /whatsnew for Firefox 78.

## Issue / Bugzilla link
#8916

## Testing
- [ ] Did I miss any strings?
- [ ] Did I pick sensible "required" strings?
- [ ] Did I pick sensible Fluent IDs and provide adequate comments in the .ftl?

### CODE REVIEW

**Comment by alexgibson** [comment]
@pmac r? We'd like to try and push this page through today as a test before we do the next /whatsnew page.

File: l10n/en/firefox/installer-help.ftl [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+installer-help-main-heading = Hmm… your download was interrupted
+installer-help-main-tagline = That happens sometimes. Let’s try something different.
+installer-help-action-channel = This download should work better for you.
+installer-help-action-choose = Choose a download below.
+
+##
+installer-help-firefox-release-title = { -brand-name-firefox-browser }
+
+# Variables:
+#   $trackers (number) - number of trackers blocked by Firefox
```
Comment (pmac) [inline l10n/en/firefox/installer-help.ftl:18]: Should we give an approximate number here? Like "currently 2000" or "will be thousands".

File: l10n/en/firefox/installer-help.ftl [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+installer-help-firefox-release-desc = Get the latest. Automatic privacy is here. Download { -brand-name-firefox } to block over { $trackers } trackers.
```
Comment (pmac) [inline l10n/en/firefox/installer-help.ftl:19]: I see this is using `-brand-name-firefox` and above uses `-brand-name-firefox-browser`. What's the difference?

File: l10n/en/firefox/installer-help.ftl [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/installer-help/
+
+## Required
+installer-help-page-title = Your download was interrupted
+installer-help-main-heading = Hmm… your download was interrupted
```
Comment (pmac) [inline l10n/en/firefox/installer-help.ftl:9]: Is this really simple English, or should it move to an en-US file?

File: l10n/en/firefox/installer-help.ftl (hunk 1 above)
Comment (alexgibson) [reply l10n/en/firefox/installer-help.ftl:18]: Good suggestion 👍

File: l10n/en/firefox/installer-help.ftl (hunk 2 above)
Comment (alexgibson) [reply l10n/en/firefox/installer-help.ftl:19]: Hmm yeah - this does seem inconsistent. Let me update to use Firefox Browser for completeness.

File: l10n/en/firefox/installer-help.ftl (hunk 3 above)
Comment (alexgibson) [reply l10n/en/firefox/installer-help.ftl:9]: Good catch - yeah let's move this to an en-US file.

**Comment by alexgibson** [comment]
Thanks @pmac - I pushed an update with some review fixes.

>  It would be nice to have someone on the l10n team also review of course.

@peiying2 can you take a look here and let us know if the .ftl file(s) look good? Thanks

**Comment by peiying2** [comment]
> @peiying2 can you take a look here and let us know if the .ftl file(s) look good? Thanks
Looks good!

**Pull request merged**